relay-event-normalization/src/event.rs

```diff
@@ -202,6 +202,13 @@ fn normalize(event: &mut Event, meta: &mut Meta, config: &NormalizationConfig) -
     // Check for required and non-empty values
     schema::SchemaProcessor.process_event(event, meta, ProcessingState::root())?;
 
+    normalize_timestamps(
+        event,
+        meta,
+        config.received_at,
+        config.max_secs_in_past,
+        config.max_secs_in_future,
+    )?; // Timestamps are core in the metrics extraction
```

Comment:
After this normalization applied, the `TimestamProcessor::process_event` does not have anything todo, does it? 

Correct me if I'm missing something in the code, but it seems like the timestamps will already be set to correct values.

Reply:
Hmm, that makes sense. Timestamps were already processed when we introduced `TimestampProcessor`, so I wonder what I'm missing. I'll have a look at it, thanks for pointing it out!

Reply:
`TimestampProcessor` is still useful: timestamp normalization (mostly clockdrift) is sometimes not enough to accept a transaction. For example, when the end timestamp is up-to-date but the start timestamp is still stale after normalizing it. I added a test case covering it in [d09b833](https://github.com/getsentry/relay/pull/2878/commits/d09b8332ec2cf24920d9403f60153352e483a50e).